api/src/main/java/jakarta/persistence/Timeout.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) 2008, 2020 Oracle and/or its affiliates. All rights reserved.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0,
+ * or the Eclipse Distribution License v. 1.0 which is available at
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR BSD-3-Clause
+ */
+
+// Contributors:
+//     Gavin King      - 3.2
+
+
+package jakarta.persistence;
+
+/**
+ * Specifies a timeout for a database request. This option is
+ * always a hint, and may be ignored by the provider.
+ *
+ * @since 3.2
+ */
+public class Timeout implements FindOption, RefreshOption, LockOption {
+    private final int milliseconds;
+
+    private Timeout(int milliseconds) {
+        this.milliseconds = milliseconds;
+    }
+
+    /**
+     * Specify a timeout in seconds.
+     * (Abbreviation of {@link #seconds(int)}.)
+     */
+    public Timeout s(int seconds) {
```

Comment:
whoops, another oversight - this (and few other methods) should be static, right now there is no way to create/get an instance of the `Timeout`

Reply:
OMG I suck. Sorry.